config/crd/bases/datasciencecluster.opendatahub.io_datascienceclusters.yaml

```diff
@@ -246,7 +246,7 @@ spec:
                               certificate:
                                 description: Certificate specifies configuration of
                                   the TLS certificate securing communications of the
-                                  for Ingress Gateway.
+                                  for gateway.
```

Comment:
should not be a change in bundle/manifests ?

Reply:
The godoc changed because the type itself is not really about "ingress". It is Istio Gateway stuff.

Reply:
It seems that `Check config and readme updates / Ensure generated files are included (pull_request) ` job is not commenting on the PR (anymore?)

https://github.com/opendatahub-io/opendatahub-operator/actions/runs/10147710041/job/28058754754#step:5:12

/cc @AjayJagan